activesupport/lib/active_support/notifications.rb

```diff
@@ -231,6 +231,13 @@ def instrument(name, payload = {})
       #   ActiveSupport::Notifications.subscribe(/render/) do |event|
       #     @event = event
       #   end
+      #
+      # Only accepts a String, Regex or empty(matches all events) event name.
+      # Raises an error if invalid event name type is passed:
+      #
+      #  ActiveSupport::Notifications.subscribe(/render/) {|*args| ...}
```

Comment:
```suggestion
      #  ActiveSupport::Notifications.subscribe(:render) {|*args| ...}
```

`/render/` will be valid since it is a regexp.